src/app/services/services.module.ts

```diff
@@ -144,9 +144,8 @@ containerService.attachToWindow(window);
 export function initFactory(): Function {
     return async () => {
         await (storageService as HtmlStorageService).init();
-        const isDev = platformUtilsService.isDev();
 
-        if (isDev || platformUtilsService.isSelfHost()) {
+        if (window.location.origin.indexOf('vault.bitwarden.com') === -1) {
```

Comment:
This needs to be protected either using the environment variable approach OR to account for malicious (or well-meaning but poorly selected) sub-domain hijacking/spoofing (by using an explicit authority test): e.g., `https://vault.bitwarden.com.badactor.compromised-site.tv`, which would still pass this test and run with production URLs in the web vault for notifications and the business portal.

Reply:
Ah good point! back to environment variables then